_posts/03-07-01-XHProf.md

```diff
@@ -0,0 +1,36 @@
+---
+title:   XHProf
+isChild: true
+anchor:  xhprof
+---
+
+## XHProf {#xhprof_title}
+
+[XHProf][xhprof] is a function-level hierarchical profiler for PHP and has a simple HTML based navigational interface.
+The raw data collection component is implemented in C (as a PHP extension). The reporting/UI layer is all in PHP. It is
+capable of reporting function-level inclusive and exclusive wall times, memory usage, CPU times and number of calls
+for each function. Additionally, it supports ability to compare two runs (hierarchical DIFF reports), or aggregate
+results from multiple runs.
+
+### How to install XHProf
+
+Then use `pecl` to install the xhprof extension:
```

Comment:
This line reads as if something was written before it and removed. Perhaps remove the word then?